schutzbot/Jenkinsfile

```diff
@@ -317,6 +328,18 @@ pipeline {
                         }
                     }
                 }
+                stage('EL8.3 OSTree') {
```

Comment:
Same question as above, why not a target for EL8 too ?

Reply:
Same. RHEL for Edge is only supported on 8.3, but EL8 uses the latest GA (8.2).